adapters/Mintegral/MintegralAdapter/GADMediationAdapterMintegralConstants.h

```diff
@@ -15,7 +15,7 @@
 #import <Foundation/Foundation.h>
 
 /// Mintegral mediation adapter version.
-static NSString *const GADMAdapterMintegralVersion = @"7.1.9.0.0";
+static NSString *const GADMAdapterMintegralVersion = @"7.2.1.0";
```

Comment:
Since the Mintegral SDK version in Cocoapods is `7.2.1.0`, can you update this to `7.2.1.0.0`?

Reply:
Seems like it's updated to `7.2.1`: https://cocoapods.org/pods/MintegralAdSDK, please ignore this comment if it does.

Reply:
> Seems like it's updated to `7.2.1`: https://cocoapods.org/pods/MintegralAdSDK, please ignore this comment if it does.

ok,we have updated to 7.2.1 and Mintegral Ad SDK version will be 3 digits.